lib/Util/Util.php

```diff
@@ -333,4 +333,14 @@ public static function normalizeId($id)
         }
         return [$id, $params];
     }
+
+    /**
+     * Returns UNIX timestamp in milliseconds
+     *
+     * @return float current time in millis
+     */
+    public static function currentTimeMillis()
+    {
+        return round(microtime(true) * 1000);
```

Comment:
it's probably should be casted to int like:
`return (int) round(microtime(true) * 1000);`